app/controllers/products_controller.rb

```diff
@@ -1,28 +1,47 @@
 class ProductsController < ApplicationController 
-    def index
-        @products= Product.all
-        @order_item = current_order.order_items.new
-    end
+	def index
```

Comment:
bad indentation?

Reply:
Visual studio looks fine.Florin told me about Rubocop and i will start to use it.